packages/tests/package.json

```
@@ -41,7 +41,8 @@
     "check:lint": "eslint src tests",
     "check:spelling": "cspell \"{README.md,{tests,src}/**/*.ts}\"",
     "check:tsc": "tsc -p tsconfig.dev.json",
-    "test": "node ./src/run-tests.js \"tests/**/!(*.optional).spec.ts\"",
+    "test": "run-s test:*",
```

It shouldn't run `npm test:optional` 
```suggestion
    "test": "npm run test:node",
```